### PR TITLE
render infinite or NaN relevance as "0.0"

### PR DESCRIPTION
**Pull request by arnej27959**
@havardpe please take a look

### CODE REVIEW

**Review by baldersheim — APPROVED** [review]
A nice and pragmatic solution....

File: container-search/src/main/java/com/yahoo/search/rendering/JsonRenderer.java [hunk 1]
```diff
@@ -488,7 +488,8 @@ private void renderHitContents(Hit hit) throws IOException {
         if (id != null)
             generator.writeStringField(ID, id);
 
-        generator.writeNumberField(RELEVANCE, hit.getRelevance().getScore());
+        generator.writeFieldName(RELEVANCE);
+        generator.writeNumber(hit.getRelevance().toString());
```
Comment (havardpe) [inline container-search/src/main/java/com/yahoo/search/rendering/JsonRenderer.java]: how will this interact with custom renderers?
do we expect them to use toString or getScore?

**Comment by havardpe** [comment]
looks good
maybe add a unit test to verify that -inf is picked up and formatted as 0.0?

File: container-search/src/main/java/com/yahoo/search/rendering/JsonRenderer.java (hunk 1 above)
Comment (havardpe) [reply container-search/src/main/java/com/yahoo/search/rendering/JsonRenderer.java]: (as in maybe we also need to change the XML renderer to unbreak the system tests)

**Pull request merged**